ruby_test.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class TestHound
+  NAME = "Wormbots"
+  VERSION = '0.0.1'
+  LOGISTICS = [NAME, VERSION]
+
+  puts "Test"
+  puts "Test another"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->